mmcv/runner/checkpoint.py

```diff
@@ -374,7 +401,26 @@ def save_checkpoint(model, filename, optimizer=None, meta=None):
         checkpoint['optimizer'] = {}
         for name, optim in optimizer.items():
             checkpoint['optimizer'][name] = optim.state_dict()
-    # immediately flush buffer
-    with open(filename, 'wb') as f:
-        torch.save(checkpoint, f)
-        f.flush()
+
+    if filename.startswith('pavi://'):
+        from pavi import modelcloud
```

Comment:
Environment may not install pavi, should allert the users to install pavi in this case